setup.py

```diff
@@ -6,7 +6,7 @@
 
 setup(
     name='pytest-xpara',
-    version='0.1.1',
+    version='0.1.2',
```

Comment:
Would it be a new minor release (`0.2.0`) since the Python 2.x will be out-of-support?

Reply:
Of course yes,